parbase/FairParamList.h

```diff
@@ -82,7 +82,7 @@ class FairParamObj : public TNamed
 class FairParamList : public TObject
 {
   protected:
-    TList* paramList;      // List for parameters stored as string
+    THashTable* paramList;      // List for parameters stored as string
```

Comment:
Does this have to be a pointer? Or can it be just `THashTable paramList;`?

Reply:
Also, this needs a version bump in the `ClassDef` macro, doesn't it?

Reply:
Theoretically yes, but as we do not save the class itself to root file it is not really needed

Reply:
Probably yes. I will check. 
I  closed the merge request since it was done against the master branch. Please comment #945

Reply:
@dennisklein  Implement the version bump of the ClassDef macro. I really forgot it. Thanks for the reminder.
@rbx  Switched from THashTable* to THashTable. Unfortunately I still have to pass a THashTable* as return value of the function getList. THashTable does not allow to return a refernce.